src/puppetlabs/puppetdb/command.clj

```diff
@@ -69,7 +69,7 @@
             [puppetlabs.puppetdb.utils :as utils]
             [slingshot.slingshot :refer [try+ throw+]]
             [puppetlabs.puppetdb.mq-listener :as mql]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.constants :as cmd-consts]
```

Comment:
I think I'd probably also lean against some of the changes just wrt the general issue of introducing potential merge conflicts (for lines that wouldn't otherwise be changing), but if there ends up being a consensus in favor, no worries.